### PR TITLE
docs: Added Cloud Seeder to other area

### PR DESCRIPTION
**Pull request by ipv6rs**
Added Cloud Seeder by IPv6.rs to the `other` area. Cloud Seeder lets you 1-click install Jellyfin on Windows, MacOS and Linux!

Thank you for the consideration!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,7 @@
 This section includes software, guides and tools that are not *specifically* designed to work with Jellyfin, but can be useful for tasks related to media management or to enhance Jellyfin's functionality.
 
 - [calibre2jellyfin](https://github.com/shawn61cp/calibre2jellyfin) - Python script to construct a Jellyfin ebook library from a Calibre library.
+- [Cloud Seeder](https://ipv6.rs/cloudseeder) - A 1-click installer and maintainer for Jellyfin on Windows, Mac and Linux.
```
Comment (darmiel) [inline README.md:86]: Hi, we would prefer to use the GitHub link instead:
```suggestion
- [Cloud Seeder](https://github.com/ipv6rslimited/cloudseeder) - A 1-click installer and maintainer for Jellyfin on Windows, Mac and Linux.
```

**Review by darmiel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ipv6rs) [reply README.md]: My apologies! I wasn't aware there was actually a button that allows us to commit a suggestion; so I went ahead and made a pull request. By the time I realized you could click a commit suggestion button, it was already too late. I hope this is alright!

Also, thank you for the suggestion as it makes more sense.

**Comment by ipv6rs** [comment]
@darmiel My apologies, I made a comment on the suggestion as well, but I was unaware there was a 'commit suggestion' button. As such I already made a commit with the change you suggested. I apologize for that, and thank you for the suggestion and for showing me there is a suggestion/commit suggestion function!

All changes are committed and this is ready for merge!

Thank you!

**Review by TomRomeo — APPROVED** [review]

**Review by darmiel — APPROVED** [review]

**Comment by ipv6rs** [comment]
Thank you!

**Pull request merged**